JitsiConference.js

```diff
@@ -3663,3 +3663,50 @@ JitsiConference.prototype.lobbyApproveAccess = function(id) {
         this.room.getLobby().approveAccess(id);
     }
 };
+
+/**
+ * Returns <tt>true</tt> if AV Moderation support is enabled in the backend.
+ *
+ * @returns {boolean} whether AV Moderation is supported in the backend.
+ */
+JitsiConference.prototype.isAVModerationSupported = function() {
+    return Boolean(this.room && this.room.getAVModeration().isSupported());
+};
+
+/**
+ * Enables AV Moderation by moderators
+ */
+JitsiConference.prototype.enableAVModeration = function() {
+    if (this.room && this.isModerator()) {
+        this.room.getAVModeration().enable(true);
+    }
+};
+
+/**
+ * Disabled AV Moderation by moderators
+ */
+JitsiConference.prototype.disableAVModeration = function() {
+    if (this.room && this.isModerator()) {
+        this.room.getAVModeration().enable(false);
+    }
+};
+
+/**
+ * Approve participant access to certain media, allows unmuting audio or video.
+ *
+ * @param {MediaType} mediaType "audio" or "video"
+ * @param id the id of the participant.
+ */
+JitsiConference.prototype.avModerationApprove = function(mediaType, id) {
+    if (this.room && this.isModerator()
```

Comment:
Should we log if these conditions (`this.room && this.isModerator()` in this function and the ones above) are not met?

Reply:
Not really ... the same is and for lobby ... or maybe we can add something so if someone is using the lib in a wrong way can see it ... ok I will add it

Reply:
Done.